modules/nodes-containers-volumes-subpath.adoc

```diff
@@ -15,7 +15,7 @@ instead of the volume's root.
 +
 ----
 $ oc rsh <pod>
-sh-4.3$ ls /path/to/volume/subpath/mount
+sh-4.4$ ls /path/to/volume/subpath/mount
```

Comment:
Same comment as above, I believe this should be reverted

Reply:
This is what I see: `sh-4.2#`
```
$ oc rsh etcd-ip-10-0-135-57.us-west-2.compute.internal
Defaulting container name to etcdctl.
Use 'oc describe pod/etcd-ip-10-0-135-57.us-west-2.compute.internal -n openshift-etcd' to see all of the containers in this pod.
sh-4.2#
sh-4.2# exit
exit
mburke@mburke ~ $ oc version
Client Version: openshift-clients-4.3.0-201910250623-42-gc276ecb7
Server Version: 4.4.0-rc.11
Kubernetes Version: v1.17.1

```